packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts

```diff
@@ -364,6 +364,9 @@ export class CamelComponentSchemaService {
         break;
 
       case 'log':
+        if (typeof definition === 'object' && !isDefined(definition.message)) {
+          updatedDefinition = { ...definition, message: '${body}' };
+        }
```

Comment:
Hi @shivamG640,  this is the right file to add the logic: https://github.com/KaotoIO/kaoto/blob/d9986e86014ef3a6640a3f9cbc1ada333622c49f/packages/ui/src/models/visualization/flows/support/camel-component-default.service.ts#L118-124

You just need to add another case with the `message` property in it